lib/ember-build.js

```diff
@@ -194,7 +195,9 @@ var EmberBuild = CoreObject.extend({
       srcFile:  this._name + '.prod.js',
       destFile: this._name + '.min.js',
       mangle:   true,
-      compress: true
+      compress: {
+        global_defs: { DEV_MODE: false }
```

Comment:
So I was going to use `isEnabled` but this forces module semantics and I don't want this to be a module, I want the helpers to be global in Ember's closure.